scripts/quarantine_stats/generate_dashboard.py

```
@@ -0,0 +1,1959 @@
+    """
+
+    def sort_key(branch_name: str) -> tuple[int, float]:
+        if branch_name == "main":
+            return (0, 0)
+        # Extract version number for cnv-X.Y branches
+        match = CNV_BRANCH_PATTERN.match(string=branch_name)
+        if match:
+            version_str = branch_name.replace("cnv-", "")
+            try:
+                version = float(version_str)
+                return (1, -version)  # Negative for descending order
```

Ruff (0.14.11)

Consider moving this statement to an `else` block

(TRY300)